jdaviz/configs/imviz/plugins/compass/compass.py

```diff
@@ -26,12 +26,15 @@ class Compass(PluginTemplateMixin, ViewerSelectMixin):
     icon = Unicode("").tag(sync=True)
     data_label = Unicode("").tag(sync=True)
     img_data = Unicode("").tag(sync=True)
+    canvas_angle = Float(0).tag(sync=True)  # set by canvas rotation plugin
+    canvas_rotatey = Integer(1).tag(sync=True)

```

Comment:
Can we rename "canvas_rotatey" to something less cryptic? I have no idea what it is supposed to do by just the name without having to read the code. Is it just me?

Reply:
I was a bit torn on that... `rotatey` is the css property that is being set, and this isn't intended to be exposed to the user.  I guess I could use the same `flip`/`flip_horizontal` boolean, but then do need logic on the JS side to map that to the CSS property, and I was trying to avoid duplicating that logic.  So... any preference between some duplicated JS code but consistent traitlets, or this somewhat cryptic traitlet name and adding an inline comment?

Reply:
I switched this to consistently use `flip_horizontal` (or `canvas_flip_horizontal` when stored outside the plugin).